commands/Minigame.js

```diff
@@ -128,6 +129,11 @@ export default class Minigame extends Command {
     this.progress(client);
   }
 
+  reactionRetr(reaction, user) {
+    if (this.#state === State.STARTING && this.#players.has(user))
+      this.#players.delete(user);
```

Comment:
```suggestion
    if (this.#state === State.STARTING) this.#players.delete(user);
```

`this.#players.has(user)` is not necessary; the map#delete method does that check internally, and ignores the key if not present:

https://tc39.es/ecma262/#sec-map.prototype.delete
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/delete